### PR TITLE
Update ci docker-compose file

### PR DESCRIPTION
**Pull request by josejulio**
@israel-hdez please take a look

### CODE REVIEW

File: ci_files/docker-compose.yml [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
+  prometheus:
+    image: prom/prometheus:v2.0.0
     environment:
       - CASSANDRA_START_RPC=true
       - MAX_HEAP_SIZE=128M
       - HEAP_NEWSIZE=24M
+    command: --config.file=/prometheus.yml
+    volumes:
+      - "hawkular_data:/var/hawkular"
+      - "./prometheus.yml:/prometheus.yml"
```
Comment (israel-hdez) [inline ci_files/docker-compose.yml]: Do we need the volumes? I guess the named volumes it's for sharing data in both containers.
But, do we need to map `prometheus.yml` locally?

Comment (josejulio) [reply ci_files/docker-compose.yml]: whoops, its the other way, we have to put a copy of prometheus.yml there, forgot to add that file.

Comment (israel-hdez) [reply ci_files/docker-compose.yml]: That makes sense :laughing:

**Comment by josejulio** [comment]
Added the Prometheus file and removed cassandra environment variables

**Review by israel-hdez — APPROVED** [review]
:+1:

**Pull request merged**